.github/workflows/release.yml

```diff
@@ -53,3 +53,5 @@ jobs:
           generateReleaseNotes: true
```

Comment:
```suggestion
```
I think removing that should be enough? Though I don't see the harm in using `omitBodyDuringUpdate` as well to be extra safe.

Reply:
Yeah, I saw this, it's probably the safer option, I'll pick that instead.